tests/cupy_tests/core_tests/test_ndarray.py

```diff
@@ -223,52 +224,95 @@ class TestNdarrayCudaInterface(unittest.TestCase):
     def test_cuda_array_interface(self):
         arr = cupy.zeros(shape=(2, 3), dtype=cupy.float64)
         iface = arr.__cuda_array_interface__
+        assert iface['version'] == 3  # bump this when the protocol is updated!
         assert (set(iface.keys()) ==
                 set(['shape', 'typestr', 'data', 'version', 'descr',
-                     'strides']))
+                     'stream', 'strides']))
         assert iface['shape'] == (2, 3)
         assert iface['typestr'] == '<f8'
         assert isinstance(iface['data'], tuple)
         assert len(iface['data']) == 2
         assert iface['data'][0] == arr.data.ptr
         assert not iface['data'][1]
-        assert iface['version'] == 2
         assert iface['descr'] == [('', '<f8')]
         assert iface['strides'] is None
+        assert iface['stream'] == 1
 
     def test_cuda_array_interface_view(self):
         arr = cupy.zeros(shape=(10, 20), dtype=cupy.float64)
         view = arr[::2, ::5]
         iface = view.__cuda_array_interface__
+        assert iface['version'] == 3  # bump this when the protocol is updated!
         assert (set(iface.keys()) ==
-                set(['shape', 'typestr', 'data', 'version',
-                     'strides', 'descr']))
+                set(['shape', 'typestr', 'data', 'version', 'descr',
+                     'stream', 'strides']))
         assert iface['shape'] == (5, 4)
         assert iface['typestr'] == '<f8'
         assert isinstance(iface['data'], tuple)
         assert len(iface['data']) == 2
         assert iface['data'][0] == arr.data.ptr
         assert not iface['data'][1]
-        assert iface['version'] == 2
         assert iface['strides'] == (320, 40)
         assert iface['descr'] == [('', '<f8')]
+        assert iface['stream'] == 1
 
     def test_cuda_array_interface_zero_size(self):
         arr = cupy.zeros(shape=(10,), dtype=cupy.float64)
         view = arr[0:3:-1]
         iface = view.__cuda_array_interface__
+        assert iface['version'] == 3  # bump this when the protocol is updated!
         assert (set(iface.keys()) ==
-                set(['shape', 'typestr', 'data', 'version',
-                     'strides', 'descr']))
+                set(['shape', 'typestr', 'data', 'version', 'descr',
+                     'stream', 'strides']))
         assert iface['shape'] == (0,)
         assert iface['typestr'] == '<f8'
         assert isinstance(iface['data'], tuple)
         assert len(iface['data']) == 2
         assert iface['data'][0] == 0
         assert not iface['data'][1]
-        assert iface['version'] == 2
         assert iface['strides'] is None
         assert iface['descr'] == [('', '<f8')]
+        assert iface['stream'] == 1
+
+
+# TODO(leofang): test PTDS
+@testing.parameterize(*testing.product({
+    'stream': (cuda.Stream.null, cuda.Stream()),
```

Comment:
In the current implementation, `runtime.streamCreate()` looks called during test case collection. Could you move it to `setUp`?

Reply:
Done in ad95878.